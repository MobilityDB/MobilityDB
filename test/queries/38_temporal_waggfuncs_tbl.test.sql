-------------------------------------------------------------------------------
-- TemporalInst
-------------------------------------------------------------------------------

SELECT numSequences(wmin(inst, interval '5 minutes'))
FROM tbl_tintinst;

SELECT numSequences(wmax(inst, interval '5 minutes'))
FROM tbl_tintinst;

SELECT numSequences(wsum(inst, interval '5 minutes'))
FROM tbl_tintinst;

SELECT numSequences(wcount(inst, interval '5 minutes'))
FROM tbl_tintinst;

SELECT numSequences(wavg(inst, interval '5 minutes'))
FROM tbl_tintinst;

-------------------------------------------------------------------------------

SELECT numSequences(wmin(inst, interval '5 minutes'))
FROM tbl_tfloatinst;

SELECT numSequences(wmax(inst, interval '5 minutes'))
FROM tbl_tfloatinst;

SELECT numSequences(wsum(inst, interval '5 minutes'))
FROM tbl_tfloatinst;

SELECT numSequences(wcount(inst, interval '5 minutes'))
FROM tbl_tfloatinst;

SELECT numSequences(wavg(inst, interval '5 minutes'))
FROM tbl_tfloatinst;

-------------------------------------------------------------------------------
-- TemporalI
-------------------------------------------------------------------------------

SELECT numSequences(wmin(ti, interval '5 minutes'))
FROM tbl_tinti;

SELECT numSequences(wmax(ti, interval '5 minutes'))
FROM tbl_tinti;

SELECT numSequences(wsum(ti, interval '5 minutes'))
FROM tbl_tinti;

SELECT numSequences(wcount(ti, interval '5 minutes'))
FROM tbl_tinti;

SELECT numSequences(wavg(ti, interval '5 minutes'))
FROM tbl_tinti;

-------------------------------------------------------------------------------

SELECT numSequences(wmin(ti, interval '5 minutes'))
FROM tbl_tfloati;

SELECT numSequences(wmax(ti, interval '5 minutes'))
FROM tbl_tfloati;

SELECT numSequences(wsum(ti, interval '5 minutes'))
FROM tbl_tfloati;

SELECT numSequences(wcount(ti, interval '5 minutes'))
FROM tbl_tfloati;

SELECT numSequences(wavg(ti, interval '5 minutes'))
FROM tbl_tfloati;

-------------------------------------------------------------------------------
-- TemporalSeq
-------------------------------------------------------------------------------

SELECT numSequences(wmin(seq, interval '5 minutes'))
FROM tbl_tintseq;

SELECT numSequences(wmax(seq, interval '5 minutes'))
FROM tbl_tintseq;

SELECT numSequences(wsum(seq, interval '5 minutes'))
FROM tbl_tintseq;

SELECT numSequences(wcount(seq, interval '5 minutes'))
FROM tbl_tintseq;

SELECT numSequences(wavg(seq, interval '5 minutes'))
FROM tbl_tintseq;

-------------------------------------------------------------------------------

SELECT numSequences(wmin(seq, interval '5 minutes'))
FROM tbl_tfloatseq;

SELECT numSequences(wmax(seq, interval '5 minutes'))
FROM tbl_tfloatseq;

SELECT numSequences(wcount(seq, interval '5 minutes'))
FROM tbl_tfloatseq;

-------------------------------------------------------------------------------
-- TemporalS
-------------------------------------------------------------------------------

SELECT numSequences(wmin(ts, interval '5 minutes'))
FROM tbl_tints;

SELECT numSequences(wmax(ts, interval '5 minutes'))
FROM tbl_tints;

SELECT numSequences(wsum(ts, interval '5 minutes'))
FROM tbl_tints;

SELECT numSequences(wcount(ts, interval '5 minutes'))
FROM tbl_tints;

SELECT numSequences(wavg(ts, interval '5 minutes'))
FROM tbl_tints;

-------------------------------------------------------------------------------

SELECT numSequences(wmin(ts, interval '5 minutes'))
FROM tbl_tfloats;

SELECT numSequences(wmax(ts, interval '5 minutes'))
FROM tbl_tfloats;

SELECT numSequences(wcount(ts, interval '5 minutes'))
FROM tbl_tfloats;

-------------------------------------------------------------------------------
