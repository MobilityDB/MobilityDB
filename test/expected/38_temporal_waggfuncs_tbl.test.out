SELECT numSequences(wmin(inst, interval '5 minutes'))
FROM tbl_tintinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wmax(inst, interval '5 minutes'))
FROM tbl_tintinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wsum(inst, interval '5 minutes'))
FROM tbl_tintinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wcount(inst, interval '5 minutes'))
FROM tbl_tintinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wavg(inst, interval '5 minutes'))
FROM tbl_tintinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wmin(inst, interval '5 minutes'))
FROM tbl_tfloatinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wmax(inst, interval '5 minutes'))
FROM tbl_tfloatinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wsum(inst, interval '5 minutes'))
FROM tbl_tfloatinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wcount(inst, interval '5 minutes'))
FROM tbl_tfloatinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wavg(inst, interval '5 minutes'))
FROM tbl_tfloatinst;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wmin(ti, interval '5 minutes'))
FROM tbl_tinti;
 numsequences 
--------------
          352
(1 row)

SELECT numSequences(wmax(ti, interval '5 minutes'))
FROM tbl_tinti;
 numsequences 
--------------
          355
(1 row)

SELECT numSequences(wsum(ti, interval '5 minutes'))
FROM tbl_tinti;
 numsequences 
--------------
          466
(1 row)

SELECT numSequences(wcount(ti, interval '5 minutes'))
FROM tbl_tinti;
 numsequences 
--------------
          466
(1 row)

SELECT numSequences(wavg(ti, interval '5 minutes'))
FROM tbl_tinti;
 numsequences 
--------------
          680
(1 row)

SELECT numSequences(wmin(ti, interval '5 minutes'))
FROM tbl_tfloati;
 numsequences 
--------------
          486
(1 row)

SELECT numSequences(wmax(ti, interval '5 minutes'))
FROM tbl_tfloati;
 numsequences 
--------------
          490
(1 row)

SELECT numSequences(wsum(ti, interval '5 minutes'))
FROM tbl_tfloati;
 numsequences 
--------------
          753
(1 row)

SELECT numSequences(wcount(ti, interval '5 minutes'))
FROM tbl_tfloati;
 numsequences 
--------------
          503
(1 row)

SELECT numSequences(wavg(ti, interval '5 minutes'))
FROM tbl_tfloati;
 numsequences 
--------------
          753
(1 row)

SELECT numSequences(wmin(seq, interval '5 minutes'))
FROM tbl_tintseq;
 numsequences 
--------------
           95
(1 row)

SELECT numSequences(wmax(seq, interval '5 minutes'))
FROM tbl_tintseq;
 numsequences 
--------------
           96
(1 row)

SELECT numSequences(wsum(seq, interval '5 minutes'))
FROM tbl_tintseq;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wcount(seq, interval '5 minutes'))
FROM tbl_tintseq;
 numsequences 
--------------
           97
(1 row)

SELECT numSequences(wavg(seq, interval '5 minutes'))
FROM tbl_tintseq;
 numsequences 
--------------
          650
(1 row)

SELECT numSequences(wmin(seq, interval '5 minutes'))
FROM tbl_tfloatseq;
 numsequences 
--------------
          119
(1 row)

SELECT numSequences(wmax(seq, interval '5 minutes'))
FROM tbl_tfloatseq;
 numsequences 
--------------
          117
(1 row)

SELECT numSequences(wcount(seq, interval '5 minutes'))
FROM tbl_tfloatseq;
 numsequences 
--------------
           98
(1 row)

SELECT numSequences(wmin(ts, interval '5 minutes'))
FROM tbl_tints;
 numsequences 
--------------
          427
(1 row)

SELECT numSequences(wmax(ts, interval '5 minutes'))
FROM tbl_tints;
 numsequences 
--------------
          424
(1 row)

SELECT numSequences(wsum(ts, interval '5 minutes'))
FROM tbl_tints;
 numsequences 
--------------
          451
(1 row)

SELECT numSequences(wcount(ts, interval '5 minutes'))
FROM tbl_tints;
 numsequences 
--------------
          447
(1 row)

SELECT numSequences(wavg(ts, interval '5 minutes'))
FROM tbl_tints;
 numsequences 
--------------
         3130
(1 row)

SELECT numSequences(wmin(ts, interval '5 minutes'))
FROM tbl_tfloats;
 numsequences 
--------------
          623
(1 row)

SELECT numSequences(wmax(ts, interval '5 minutes'))
FROM tbl_tfloats;
 numsequences 
--------------
          605
(1 row)

SELECT numSequences(wcount(ts, interval '5 minutes'))
FROM tbl_tfloats;
 numsequences 
--------------
          475
(1 row)

